### PR TITLE
ENH: add support for "same" in all ops

### PR DESCRIPTION
**Pull request by npinto**

### CODE REVIEW

**Comment by npinto** [comment]
bump

**Comment by poilvert** [comment]
let me know if you think it is good to merge.

File: sthor/util/pad.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+# Authors: Nicolas Pinto <nicolas.pinto@gmail.com>
+#          Nicolas Poilvert <nicolas.poilvert@gmail.com>
+#
+# License: BSD
+
+import numpy as np
+
+
+def pad(arr_in, neighborhood, pad_val=0.):
```
Comment (npinto) [inline sthor/util/pad.py]: should be renamed to pad2d, i.e. pad the first 2 dimensions.

Comment (poilvert) [inline sthor/util/pad.py]: On Thu, Mar 29, 2012 at 4:50 PM, Nicolas Pinto <
reply@reply.github.com

> wrote:
> 
> > @@ -0,0 +1,61 @@
> > +"""Padding of a 3D array along the first two dimensions"""
> > +
> > +# Authors: Nicolas Pinto nicolas.pinto@gmail.com
> > +#          Nicolas Poilvert nicolas.poilvert@gmail.com
> > +#
> > +# License: BSD
> > +
> > +import numpy as np
> > +
> > +
> > +def pad(arr_in, neighborhood, pad_val=0.):
> 
> should be renamed to pad2d, i.e. pad the first 2 dimensions.

done

> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/nsf-ri-ubicv/sthor/pull/4/files#r623172

## 

Nicolas POILVERT, PhD
Post-doctoral Fellow
Rowland Institute at Harvard

**Comment by npinto** [comment]
Just missing some quick tests and we can merge.

**Comment by poilvert** [comment]
added a few unit tests for `pad.py` module

**Comment by npinto** [comment]
Looks good to me. Merging now.

**Pull request merged**